software/openvisualizer/openvisualizer/moteProbe/moteProbe.py

```diff
@@ -63,11 +63,25 @@ def findSerialPorts():
         else:
             portMask = '/dev/ttyUSB*'
         serialports = [(s,BAUDRATE_GINA) for s in glob.glob(portMask)]
+
+    # An OpenWSN mote normally sends a lot of data with \x7e as frame delimiter,
+    # so this should find all active motes
+    mote_ports = []
+
+    for port in serialports:
+        for baudrate in [port[1], 500000]:
+            with serial.Serial(port[0],baudrate,timeout=0.3) as probe_serial:
+                probe_serial.setDTR(0)
+                probe_serial.setRTS(0)
+
+                rxBytes = probe_serial.read(60)
+                if len(rxBytes) == 60 and rxBytes.count('\x7e') > 4:
```

Comment:
Just checking for one \x7e seemed too unsafe for me.
One could guess the number of bytes received within the timeout duration and the number of \x7e from the packet rate, but that does not seem more reliable than some arbitrary numbers.
The question is: Can we rely on the fact that there will always be packets sent?